server/utils/get-metadata-from-request.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { buildApiUrl } from '@sttm/banidb';
+
+import { API_URL } from '../../common/constants';
+
+/**
+ * @param {object} req - The request obj
+ *
+ */
+export const getMetadataFromRequest = (req) => {
+  const { path, query } = req;
+  switch (path) {
+    case '/shabad': {
+      const _url = buildApiUrl({ id: query.id, API_URL });
+      const url = 'https:' + _url;
+      return axios.get(url);
+    }
+    // We are returni ng a resolved promise here with falsy value
+    // to indicate we get empty data from api.
+    default: return Promise.resolve(null);
+  }
+}
```

Comment:
Consider using async/await syntax instead.

Reply:
@bogas04 I didn't add async/await since at hosting side we are using node.js version 8 thought it might can cause issues there.

Reply:
Node 8 supports it well

Reply:
:+1: thanks

Reply:
@bogas04 added that in the main express router.